confidence/io.py

```diff
@@ -305,61 +313,70 @@ def load_name(
         significance
     :param load_order: ordered list of name templates or `callable` s, in
         increasing order of significance
-    :param extension: file extension to be used
```

Comment:
missing the extension docstring parameter?

Reply:
That was removed on purpose as it was also deprecated (in favour of the new format argument). Keeping the doc around seemed wrong, but it might make sense for as long as it's not getting actually removed? 🤔